testnet/src/config.rs

```diff
@@ -39,6 +39,66 @@ impl ConfigFile {
     pub fn from_str(content: &str) -> ConfigFile {
         toml::from_slice(&content.as_bytes()).unwrap()
     }
+
+    pub fn neon() -> ConfigFile {    
+        let burnchain = BurnchainConfigFile {
+            mode: Some("neon".to_string()),
+            commit_anchor_block_within: Some(40_000),
+            rpc_port: Some(18443),
+            peer_port: Some(18444),
+            peer_host: Some("35.245.58.246".to_string()),
```

Comment:
Is this IP address stable?

Reply:
One thing we could do is have a DNS name for the leader node, and just resolve it to an IP address when the neon node boots up.

Reply:
+1 to DNS. To push this even further, we should try to get this config to a state (or, instead a set of default config files) that it doesn't require changing every time we reboot the Neon node, or have to switch VMs etc.

Reply:
+1 for DNS

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1464/commits/56b28a981f10f2cd83d9668e3166c14c6b7e19a2